packages/core-manager/package.json

```diff
@@ -22,6 +22,7 @@
     },
     "dependencies": {
         "argon2": "^0.26.2",
+        "axios": "^0.19.2",
```

Comment:
We use https://github.com/sindresorhus/got instead of axios in all our projects.

Reply:
Ok. I was looking for something that you use. Do you have any existing wrapper around? If not, then I will use existing HttpClient class.

Reply:
Not sure why you would need a wrapper? Just can simply call `got(url)` and use https://github.com/nock/nock to mock the responses.

Reply:
For wishes like this :) Changes or replacements of 3rd party libraries are then handled only in once place.